CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+## Contributors guidelines
+ 
+### Getting started
+To start contributing to the Onfido Web SDK, clone this repository and run the following commands:
+ 
+- `npm install` to install the project dependencies
+- `npm run dev` to generate the build and run the dev server
+ 
+The SDK will be running on https://localhost:8080.
+ 
+Note: The SDK runs on `https`, which is needed to support the camera functionality.
+ 
+You can also run it using `docker` and `docker-compose` by running the following command: `docker-compose -f docker-compose.dev.yml`.
+ 
+### Testing
+The Web SDK has a large coverage of UI tests. To set up your environment, please refer to the [testing guidelines](./test/TESTING_GUIDELINES.md).
+Once your environment is ready, you can run UI tests by running `npm run build:test && npm run travis && npm run test:ui`.
+The Web SDK has partial coverage of unit tests that can be executed with `npm run test`.
+You can also run a type checker and linter by running `npm run check`.
+ 
+### Contributing
+When creating a new branch, contributors should use the following convention `{task-type}/{task-description}-{ticket-number}`.
```

Comment:
I thought we said we don't include Jira ticket number in the branch name. Has that changed?

Reply:
I don't remember having this conversation but I'm happy to change this if that's what we agreed on

Reply:
Same. I don't remember that conversation either in terms of the branch naming. 
But wouldn't knowledge of a ticket number depend on whether the contributor is an internal Onfido staff? Perhaps issue number for external contributors, if applicable?

Reply:
Yes, it's only for internal contributors, that's why I added `The ticket number is optional and is only expected from internal contributors.`. For external contributors I don't see much benefit in adding the issue number in the branch name but it would definitely be beneficial to reference it in the PR description. What do you think @kopijunkie @bartzet?

Reply:
https://github.com/onfido/onfido-sdk-ui/pull/920#issuecomment-588927096

I remembered this comment ☝️ and I think once @rfreitas suggested avoiding adding them to branch names.

Reply:
Ah yes, in the comment I was referring to actual links and ticket numbers in the description. We can remove them but I don't think having a ticket number in the branch name is a big concern.

Reply:
I personally have decided against including the JIRA ticket number in the PR title also as I think it's not necessary and lacks context for anyone external looking at it. But I agree that not having them in the branch name could make it hard to keep track of PRs in relation to tickets unless we're very strict about linking PRs in our JIRA tickets & there is an easy way to figure out what JIRA ticket a PR belongs to from Github end.

Reply:
Just to wrap up, I think it’s outside the scope of this ticket to define the process, in this story we are just meant to document it. Currently some of us use tickets numbers in branched. If we want to revisit this process we can take the conversation on our private channel.
I have reworded this paragraph as
>The most used task types are `feature`, `fix` or `improvement`. The ticket number is an optional reference to our internal ticketing system.
If your pull request is related to a GitHub issue, please reference the issue in your PR description.